pkg/schedule/filter/region_filters.go

```diff
@@ -164,3 +166,29 @@ func (f *regionWitnessFilter) Select(region *core.RegionInfo) *plan.Status {
 	}
 	return statusOK
 }
+
+// SnapshotSenderFilter filer the region who's leader store reaches the limit.
+type SnapshotSenderFilter struct {
+	senders map[uint64]struct{}
+}
+
+// NewSnapshotSendFilter returns creates a RegionFilter that filters regions with witness peer on the specific store.
+// level should be set as same with the operator priority level.
+func NewSnapshotSendFilter(stores []*core.StoreInfo, level constant.PriorityLevel) RegionFilter {
+	senders := make(map[uint64]struct{})
+	for _, store := range stores {
```

Comment:
We just need to calculate whether the storeLimit of the store where the leader is located is available.  Does it seem no need to check all store?

Reply:
In some case，the scheduled region maybe pick success by may times.